ddl/cluster.go

```diff
@@ -263,31 +417,39 @@ func (w *worker) onFlashbackCluster(d *ddlCtx, t *meta.Meta, job *model.Job) (ve
 		return ver, nil
 	// Stage 2, check flashbackTS, close GC and PD schedule.
 	case model.StateWriteOnly:
-		sess, err := w.sessPool.get()
-		if err != nil {
-			job.State = model.JobStateCancelled
-			return ver, errors.Trace(err)
-		}
-		defer w.sessPool.put(sess)
 		if err = checkAndSetFlashbackClusterInfo(sess, d, t, job, flashbackTS); err != nil {
 			job.State = model.JobStateCancelled
 			return ver, errors.Trace(err)
 		}
+		// A hack way to make global variables are synchronized to all tidb.
+		// TiKV will block read/write request during flashback cluster,
+		// So it's not very dangerous when sync failed.
+		time.Sleep(1 * time.Second)
```

Comment:
Why here is `1s`?

Reply:
You can see the comments, just let most TiDB synced this global variable.

Reply:
`1s` just a magic number. I found the global variable will synced by ETCD, so the time cost will be very small

Reply:
Got it.